README.md

```diff
@@ -12,11 +12,46 @@ and [Liberty Gradle](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/l
 
 Note that this extension requires the [Integrated Terminal plugin](https://plugins.jetbrains.com/plugin/13123-terminal) to be enabled.
 
+## Quick Start
+
+- Search for _Open Liberty Tools_ in IntelliJ **Settings > Plugins > Marketplace** and click the Install button.
+- Open Liberty supported projects will appear in the Liberty Dev Dashboard on the side bar
+- Click a project in the Liberty Dev Dashboard to view the available commands
+
+## Features
+
+- View supported `liberty-maven-plugin`(version `3.1` or higher) or `liberty-gradle-plugin`(version `3.0` or higher) projects in the workspace
+- Start/Stop dev mode
+- Start dev mode with custom parameters
+- Run tests
+- View unit and integration test reports
+
+## Commands
+
+| Command                      | Description                                                                                                                                                                                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Start                        | Starts dev mode.                                                                                                                                                                                                                                                                                                             |
+| Start…​                      | Starts dev mode with custom parameters. Supported parameters can be found in the documentation for the [dev goal of the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev) and the [libertyDev task of the Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#command-line-parameters). |
```

Comment:
If the intent is to link to the parameters sections of the docs, use `https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#additional-parameters`